#!/usr/bin/env python
PACKAGE = "monitoring"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("round_detection_type",       int_t, 0, "0: Startline, 1: Time",  0, 0, 1)
gen.add("round_detection_time",     double_t, 0, "time for completing one round",  0, 0, 100)

exit(gen.generate(PACKAGE, "fitness_monitor", "FitnessMonitor"))
